platform-sdk/swirlds-unit-tests/common/swirlds-common-test/build.gradle.kts

```diff
@@ -15,36 +15,37 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
-  id("org.gradle.java-test-fixtures")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
```

Comment:
intentional?

Reply:
Looks like spotless did it since it was changed for the first time?